mne/inverse_sparse/tests/test_mxne_optim.py

```diff
@@ -120,6 +127,15 @@ def test_tf_mxne():
     assert_array_less(gap_tfmxne, 1e-8)
     assert_array_equal(np.where(active_set_hat_tf)[0], active_set)
 
+    alpha_space = 1e8
+    alpha_time = 1e8
```

Comment:
Could we have it with alpha_space = 1.01 * alpha_max * (1. - l1_ratio) and alpha_time = 1.01 * alpha_max * l1_ratio ?

Reply:
I wanted to avoid computing alpha_max here to have a faster test. When we want to change it i'd compute alpha_max at the beginning of the test so that we can specify alpha_space and alpha_time in both tfmxne computations based on alpha_max.

Reply:
Then we could merge this test with test_dgapl21l1() which already computes an alpha_max on some random tf data ?

Reply:
otherwise alpha_max should be quick to compute on this toy dataset